src/ansys/acp/core/_tree_objects/imported_solid_model.py

```diff
@@ -354,9 +354,14 @@ def _create_stub(self) -> imported_solid_model_pb2_grpc.ObjectServiceStub:
         layup_mapping_object_pb2_grpc.ObjectServiceStub,
     )
 
-    def refresh(self, path: _PATH, format: SolidModelImportFormat) -> None:  # type: ignore
-        """Re-import the solid model from the external file."""
-        self.format = format
+    def refresh(self, path: _PATH, format: SolidModelImportFormat | None = None) -> None:  # type: ignore
+        """
+        Re-import the solid model from the external file.
+
+        The ``format`` parameter is only used to switch it. Otherwise, the current value is used.
```

Comment:
I think this docstring should have a `Parameters` section (same for the CADGeometry, I guess I missed that..).